### PR TITLE
Update Governance Section Sidebar To Correctly Display All Files

### PR DESCRIPTION
**Pull request by thenic95**
Hi,

I have update the Governance Section `sidebar.js` to correctly display all files. 

I have missed this in PR #1284 

Best,
Nicolas

### CODE REVIEW

**Review by rphair — APPROVED** [review]
aha; building this locally I can see your new articles now... don't know how we missed this before 😅

(**p.s.** we didn't miss it; at least not in #1284... your section updates were overwritten by the recently merged https://github.com/cardano-foundation/developer-portal/pull/1287)

**Comment by thenic95** [comment]
I blame @katomm 😄 

Thank you, @rphair, I knew that I would have caught that when I built the site 🙌

File: sidebars.js [hunk 1]
```diff
@@ -132,8 +132,16 @@ module.exports = {
       "smart-contracts/opshin",
       "smart-contracts/plu-ts",
     ],
-    "Be Part of the Governance": [
+    "Participate in Governance": [
       "governance/overview",
```
Comment (rdlrt) [inline sidebars.js:136]: Did we want to remove this? as it's already in Cardano Governance sub menu?

**Comment by katomm** [comment]
Off topic: We need to leverage more the automated generated sidebars like we do already on for example https://cardano.org/docs/components/

File: sidebars.js (hunk 1 above)
Comment (rphair) [reply sidebars.js:136]: @rdlrt where are you seeing it duplicated?  When this PR builds locally it shows only `Overview` and `Governance Actions` in the `Cardano Governance` submenu.

**Review by katomm — APPROVED** [review]

File: sidebars.js (hunk 1 above)
Comment (rdlrt) [reply sidebars.js:136]: Ah - my bad, I got confused with governance/overview and governance/cardano-governance/overview indeed

**Review by rdlrt — APPROVED** [review]

**Comment by katomm** [comment]
I can not confirm this is fixed or the script that gets CIP and creates the menu has some side effects on this.

**Comment by rphair** [comment]
@katomm as of my https://github.com/cardano-foundation/developer-portal/pull/1290#issuecomment-2275282840 (and there have been no further changes) I saw both the restored menu items and the built & populated CIP submenu in my local build of this PR.

Now that it's merged I can also see these things after building from my local, updated `staging` branch.

**Comment by katomm** [comment]
I wasn't able to look into this yet, but the changes are already on production and there the site has no sidebar for the new Governance pages like: https://developers.cardano.org/docs/governance/cardano-governance/governance-model/

Interestingly now staging build is broken, but looks unrelated.

**Pull request merged**